src/overcooked_ai_py/agents/agent.py

```diff
@@ -319,7 +319,7 @@ def action(self, state):
 
             # NOTE: Assumes that calls to the action method are sequential
             self.prev_state = state
-        return chosen_action, {"action_probs": action_probs}
+        return chosen_action, {"action_probs": [action_probs]}
```

Comment:
I don't like incinsistency between RandomAgent and GreedyHumanModel action method output.
Example code:
```
from overcooked_ai_py.agents.agent import GreedyHumanModel, RandomAgent
from overcooked_ai_py.mdp.layout_generator import LayoutGenerator
from overcooked_ai_py.agents.benchmarking import AgentEvaluator

mdp_gen_params = {"layout_name": 'cramped_room'}
mdp_fn = LayoutGenerator.mdp_gen_fn_from_dict(mdp_gen_params)
env_params = {"horizon": 10}
agent_eval = AgentEvaluator(env_params=env_params, mdp_fn=mdp_fn)

trajectory = agent_eval.evaluate_human_model_pair(num_games=1)
state = trajectory["ep_states"][0][5]
grid = trajectory["mdp_params"][0]["terrain"]
greedy = GreedyHumanModel(mlam=agent_eval.env.mlam)
random = RandomAgent()
greedy.set_agent_index(0)
greedy_probs = greedy.action(state)[1]["action_probs"]
random_probs = random.action(state)[1]["action_probs"]
print("greedy agent probs:", greedy_probs, type(greedy_probs), "random_agent_probs:", random_probs, type(random_probs))
```
Output:
```
greedy agent probs: [array([0., 0., 1., 0., 0., 0.])] <class 'list'> random_agent_probs: [0.2 0.2 0.2 0.2 0.2 0. ] <class 'numpy.ndarray'>
```

Reply:
That is strange. I had the opposite problem earlier, but I will revert it back, now that I see this,